src/Mod/PartDesign/Gui/TaskChamferParameters.cpp

```diff
@@ -194,6 +198,31 @@ void TaskChamferParameters::clearButtons(const selectionModes notThis)
     DressUpView->highlightReferences(false);
 }
 
+// By adding the edges to the Gui.Selection in python, then onSelectionChanged() gets called
+// where the edges get added to the list in the task dialog and highlighted in the 3d view
+
+void TaskChamferParameters::onSelectAllEdges(void)

```

Comment:
TaskChamferParameters::onSelectAllEdges and TaskFilletParameters::onSelectAllEdges are almost identical. The only difference is the different cast but because only the Base property is accessed that belongs to PartDesign::DressUp the whole function can be moved to the base class TaskDressUpParameters. This will avoid code duplication.